exotic-attacks/activities/defaced-website/README.md

```diff
@@ -0,0 +1,24 @@
+# Name: Defaced Website
```

Comment:
Please change the name according to this convention. [1] 

[1]: https://docs.google.com/document/d/1qejQpcb4K5WoJaga0SJa78AxNh5F_Pjea2rX6DeZmIc/edit#heading=h.5q0d6z6p8fkq